### ISSUE TITLE
The construction of a HashSet in the readme is wrong

### ISSUE THREAD

**Issue by Roenbaeck** [ritelabs/ritehash#1]
The readme states that a type for a HashSet can be made as: 
```rust
pub type FxHashSet<K, V> = HashSet<K, V, FxBuildHasher>;
```
I believe it should be:
```rust
pub type FxHashSet<T> = HashSet<T, FxBuildHasher>;
```

**Comment by PsiACE** [comment]
thanks, i will fix it

**Comment by PsiACE** [comment]
fixed, and I release v0.2.0 for rust 2021 edition. thanks again
